traffic_ops/testing/api/v4/tc-fixtures.json

```diff
@@ -907,7 +907,7 @@
             "regexRemap": null,
             "regionalGeoBlocking": false,
             "remapText": null,
-            "routingName": "cdn",
+            "routingName": "ccr-ds1",
```

Comment:
Why set it to an invalid value? What scenario are we trying to test?

Reply:
That value is not invalid, what it was set to *before* was actually hiding an error. Changing this was part of the fix for #7094.

Reply:
oh.. ok

Reply:
No, it seems we don't